infra/lib/auth/stack.ts

```diff
@@ -260,7 +260,6 @@ class AuthIntegrationTestStackEnvironment extends IntegrationTestStackEnvironmen
     const customSenderKmsKey = new kms.Key(this, "CustomSenderKey", {
       description: `Key for encrypting/decrypting SMS messages sent from ${this.name} user pool`,
       removalPolicy: RemovalPolicy.DESTROY,
-      alias: this.name,
```

Comment:
Q: don't we need the alias anymore?

Reply:
This was causing my deployment to fail. Unsure what the cause was, but don't see the value in having defined. Also when this lifted to the gen 2 stack, it was dropped.